src/stdlib/GlobalUtilities.ts

```diff
@@ -39,3 +39,23 @@ export const GetInterface = new Callable("GetInterface", {
         return object.interfaces.get(ifname.value.toLowerCase()) || BrsInvalid.Instance;
     },
 });
+
+export const FindMemberFunction = new Callable("FindMemberFunction", {
+    signature: {
+        args: [
+            new StdlibArgument("object", ValueKind.Object),
+            new StdlibArgument("funname", ValueKind.String),
+        ],
+        returns: ValueKind.Interface,
+    },
+    impl: (interpreter, object: BrsComponent, funname: BrsString): BrsInterface | BrsInvalid => {
+        let iface: BrsType = BrsInvalid.Instance;
+        object.interfaces.forEach(interfaceName => {
+            if (interfaceName.methodNames.has(funname.value.toLowerCase())) {
```

Comment:
I assume the method name on the interface is stored lowercase when registering?

Reply:
yes: https://github.com/sjbarag/brs/blob/master/src/brsTypes/components/BrsComponent.ts#L28